lib/collection/src/collection/mod.rs

```diff
@@ -580,6 +580,10 @@ impl Collection {
             }
         }
 
+        for transfer in self.get_related_transfers(peer_id).await {
+            self.abort_shard_transfer(transfer.key(), None).await?;
```

Comment:
Edge case: What if you read about a transfer and it finished just before you try to delete it. Has happened in previous test failure. https://github.com/qdrant/qdrant/actions/runs/14967895398/job/42041945647?pr=6507

Reply:
It's fine, it'll be ignored deeper down: https://github.com/qdrant/qdrant/blob/9c723aa48083a4bc5e2d9124b827405b2c35b398/lib/collection/src/collection/shard_transfer.rs#L331-L333